automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -115,7 +115,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
           val request = ClusterRequest(machineConfig = Option(MachineConfig(
             // need at least 2 regular workers to enable preemptibles
             numberOfWorkers = Option(2),
-            numberOfPreemptibleWorkers = Option(10)
+            numberOfPreemptibleWorkers = Option(1)
```

Comment:
May make the test more sensitive to preemptible issues if there's only one

Reply:
i think having more preemptibles makes it more likely that one of them will get yanked

Reply:
Yeah this theory also makes sense